scripts/convert_lit_checkpoint.py

```diff
@@ -0,0 +1,238 @@
+import contextlib
+import gc
+import sys
+from functools import partial
+from pathlib import Path
+from typing import Optional, Literal, Dict, Union
+
+import torch
+
+# support running without installing as a package
+wd = Path(__file__).parent.parent.resolve()
+sys.path.append(str(wd))
+
+from lit_gpt import Config
+from lit_gpt.utils import lazy_load, incremental_save, NotYetLoadedTensor
+from scripts.convert_hf_checkpoint import load_param, layer_template
+
+
+def copy_weights_gpt_neox(
```

Comment:
The untested functions shouldnt be added in this PR, instead add them with their corresponding PRs

Reply:
removed in [4becaa5](https://github.com/Lightning-AI/lit-gpt/pull/273/commits/4becaa52afc748ca1572ea5b38e9439bb8c2781b)